### PR TITLE
Update tf version in check-terraform-syntax with renovate

### PR DESCRIPTION
**Pull request by berendt**
Since only one Terraform version is tested at a time, it is not
necessary to use the strategy matrix.

Signed-off-by: Christian Berendt <berendt@23technologies.cloud>

### CODE REVIEW

**Review by garloff — COMMENTED** [review]
I can not really review this, sorry!
Maybe we find another reviewer (@joshmue ?) or we need to do a session together.

File: .github/renovate.json [hunk 1]
```diff
@@ -2,5 +2,16 @@
...
   ],
-  "commitBody": "Signed-off-by: Renovate Bot <bot@renovateapp.com>"
+  "commitBody": "Signed-off-by: Renovate Bot <bot@renovateapp.com>",
+  "regexManagers":[
+    {
+      "fileMatch":[
+        "^\\.github\\/workflows\\/.*\\.yml"
+      ],
+      "matchStrings":[
+        "terraform_version: (?<currentValue>.*?)  # renovate: datasource=(?<datasource>.*?) depName=(?<depName>.*?)\n"
```
Comment (joshmue) [inline .github/renovate.json:12]:
```suggestion
        "terraform_version: (?<currentValue>.*?)\\s*# renovate: datasource=(?<datasource>.*?) depName=(?<depName>.*?)\n"
```

Just a very minor tweak to not require exactly two whitespaces, but any number of them. Did no testing, but should work.

Comment (joshmue) [reply .github/renovate.json:12]: Looks fine to me, otherwise!

**Review by garloff — APPROVED** [review]
Trusting Joshua's review!
THANKS!

**Pull request merged**